exporter/collector/metricsexporter.go

```diff
@@ -129,15 +129,17 @@ func (me *metricsExporter) pushMetrics(ctx context.Context, m pdata.Metrics) err
 			for k := 0; k < mes.Len(); k++ {
 				metric := mes.At(k)
 				// TODO - check to see if this is a service/system metric and doesn't send descriptors.
-				if !me.cfg.MetricConfig.SkipCreateMetricDescriptor {
-					for _, md := range me.mapper.metricDescriptor(metric) {
-						if md != nil {
-							select {
-							case me.mds <- md:
-							default:
-								// Ignore drops, we'll catch descriptor next time around.
-							}
-						}
+				if me.cfg.MetricConfig.SkipCreateMetricDescriptor {
+					continue
```

Comment:
This shouldn't be continuing here.  The timeseries still needs to be appended first.

You could move L145 above the if.

Reply:
Good catch. Done.